storage-mirror-firestore/stress_test/bin/driver.ts

```diff
@@ -711,6 +711,135 @@ const backfillCmd = new commander.Command("backfill")
     spinner.succeed(`Mirrored ${succeeded} paths in ${elapsed}s.`);
   });
 
+const cleanTombstonesCommand = new commander.Command("clean-tombstones")
```

Comment:
is the `stress_test` folder the right place for this?

Reply:
Probably not, but the other scripts like backfill for some reason were implemented here - didn't want too many changes for this PR - we could look at extracting all these commands out into it's own package in another PR if you're happy for us to do that?

Reply:
sounds good to me